akka-bbb-apps/src/main/scala/org/bigbluebutton/core/running/MeetingActor.scala

```diff
@@ -670,6 +674,69 @@ class MeetingActor(
     )
   }
 
+  private def handleGetMeetingInfo(): MeetingInfo = {
```

Comment:
`MeetingActor.scala` is becoming too large. Could you create a new class (trait) to place the handlers related to gRPC messages?